src/app/my-movies/my-movies.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from '@angular/core';
+import { NgForm } from '@angular/forms';
+
+@Component({
+  selector: 'app-my-movies',
+  templateUrl: './my-movies.component.html',
+  styleUrls: ['./my-movies.component.css']
+})
+export class MyMoviesComponent implements OnInit {
+
+  message = new TemplateMessage();
+  sorts = ['- - -', 'Najstarsze filmy', 'Ostatnio dodane filmy'];
```

Comment:
raczej nie dawaj takiego czegoś jak --- jako wartośc sorta, potem ktoś to kliknie i w funkcji sortującej będziesz stawał na chuju żeby ifami nie reagować na taki wybór; imo da się jakoś w selectach ustawić żeby domyślnie byłlo wybrane nic, a  jak za ciężko to trudno, po prostu niech defaultowa wartość sortu będzie tu pierwsza